packages/boilerplates/Scripts.ts

```diff
@@ -0,0 +1,165 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import spawn from 'cross-spawn';
+import lodash from 'lodash';
+import path from 'path';
+import { Script, ScriptFailedError } from './internal';
+
+
+/**
+ * Represents a Boilerplate's scripts
+ */
+export class Scripts
+{
+    static fromJson(scripts: any): Scripts {
```

Comment:
Is this a public method? (Unfamiliar with TypeScript syntax) - if so, it lacks JSDoc

Reply:
@einari  Methods and properties are by default public